src/main/java/Exceptions/UserNotFoundException.java

```diff
@@ -0,0 +1,29 @@
+package Exceptions;
+
+import java.lang.RuntimeException;
+import org.springframework.http.HttpStatus;
+import Exceptions.APIException;
+
+public class UserNotFoundException extends RuntimeException implements APIException {
+
+    private static final HttpStatus HTTP_STATUS = HttpStatus.NOT_FOUND;
+    private static final String message = "Invalid Username";
```

Comment:
Usual convention for static final variable name is all caps like ERROR_MESSAGE. Same at other places.

Reply:
Done